msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -28,6 +28,17 @@ Resource-specific consent (RSC) is a Microsoft Teams and Microsoft Graph API int
 |TeamsTab.ReadWrite.Group|Update this team's tabs.​|
 |TeamsTab.Delete.Group|Delete this team's tabs.​|
 |TeamMember.Read.Group|Get this team's members.​|
+| ChatSettings.Read.Chat         | Get this chat's settings.                                    |
+| ChatSettings.ReadWrite.Chat    | Update this chat's settings.                          |
+| ChatMessage.Read.Chat          | Get this chat's messages.                                    |
+| ChatMember.Read.Chat           | Get this chat's members.                                     |
+| Chat.Manage.Chat               | Manage this chat.                                             |
+| TeamsTab.Read.Chat             | Get this chat's tabs.                                        |
+| TeamsTab.Create.Chat           | Create tabs in this chat.                                     |
+| TeamsTab.Delete.Chat           | Delete this chat's tabs.                                      |
+| TeamsTab.ReadWrite.Chat        | Manage this chat's tabs.                                      |
+| TeamsAppInstallation.Read.Chat | Get which apps are installed in this chat.                   |
+| OnlineMeeting.ReadBasic.Chat   | Get basic properties of a meeting associated with this chat. |
 
```

Comment:
Can we add See Also links for the Chat RSC and Teams RSC sections in the Graph permissions page.